requests-for-comments/2023-10-31-onboarding-emails.md

```diff
@@ -22,50 +22,52 @@ This campaign triggers when a user completes the `user signed up` event for the
 3. Check users' role in organization, using the `role_at_organization` property.
     1. If `founder` then wait 12 hours, and send Joe's Newsletter Invite email.
         1. Check if user clicked the subscription CTA.
-            1. If yes, add `newsletter_cta_clicked: true` to user.
-            2. If no, do nothing. 
+            1. If `Yes`, add `newsletter_cta_clicked: true` to user.
+            2. If `No`, do nothing. 
         2. Wait 12 hours
     2. If `engineering` then wait 12 hours, and send Andy's Newsletter Invite email. 
         1. Check if user clicked the subscription CTA.
-            1. If yes, add `newsletter_cta_clicked: true` to user.
-            2. If no, do nothing. 
+            1. If `Yes`, add `newsletter_cta_clicked: true` to user.
+            2. If `No`, do nothing. 
         2. Wait 12 hours
     3. If `product` then wait 12 hours. 
         1. Check if the user has performed `action created`
-            1. If yes, do nothing.
-            2. If no, send Advice for Product Teams email.
+            1. If `Yes`, send NEW email promoting the user interview survey template
```

Comment:
<img width="666" alt="Screenshot 2023-11-09 at 13 21 22" src="https://github.com/PostHog/meta/assets/84011561/e91d1479-3053-4667-a118-b9d9ed24c828">

Reply:
- Should the comma be after "from PostHog" so it's "I'm Joe from PostHog, and..."?
- Mention calendar booking link instead of public calendar?

Reply:
<img width="569" alt="Screenshot 2023-11-09 at 16 27 39" src="https://github.com/PostHog/meta/assets/84011561/e22b8f6b-3be4-41f1-a9f3-3ffa08bbd2b1">